opensearch-operator/pkg/builders/cluster.go

```diff
@@ -47,15 +47,15 @@ func NewSTSForNodePool(
 	availableRoles := []string{
 		"master",
```

Comment:
We probably need to add `cluster_manager` for OpenSearch v2.x support.

Reply:
added